service/handlers.go

```diff
@@ -90,6 +90,9 @@ func TokenHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 	w.Header().Set("X-CSRF-Token", csrf.Token(r))
 
+	// Check the JWT and return it in the authorization header, if valid
+	JWTCheck(w, r)
```

Comment:
What happens if this returns an error? Shouldn't this return some kind of http error in that case?